cdp_scrapers/legistar_utils.py

```diff
@@ -679,7 +678,9 @@ def get_person(self, legistar_person: Dict) -> Person:
                 email=stripped(legistar_person[LEGISTAR_PERSON_EMAIL]),
                 external_source_id=legistar_person[LEGISTAR_PERSON_EXT_ID],
                 name=stripped(legistar_person[LEGISTAR_PERSON_NAME]),
-                phone=stripped(legistar_person[LEGISTAR_PERSON_PHONE]),
```

Comment:
This change is not necessary. It's my bad; I may have forgotten to mention this [PR](https://github.com/CouncilDataProject/cdp-scrapers/pull/17) that handles this.

Reply:
No, I remember you told this. I just wanted to have a cleaner look of all the data and I just added it. As it is part of your PR, I will just remove them and push the changes